x/multisig/model.go

```diff
@@ -0,0 +1,71 @@
+package multisig
+
+import (
+	"github.com/iov-one/weave"
+	"github.com/iov-one/weave/orm"
+)
+
+const (
+	// BucketName is where we store the contracts
+	BucketName = "contracts"
+	// SequenceName is an auto-increment ID counter for contracts
+	SequenceName = "id"
+)
+
+// enforce that Contract fulfils desired interface compile-time
+var _ orm.CloneableData = (*Contract)(nil)
+
+// Validate enforces sigs and threshold boundaries
+func (c *Contract) Validate() error {
+	if len(c.Sigs) == 0 {
+		return ErrMissingSigs()
+	}
+	if c.ActivationThreshold < 0 || int(c.ActivationThreshold) > len(c.Sigs) {
+		return ErrInvalidActivationThreshold()
+	}
+	if c.ActivationThreshold < 0 {
```

Comment:
This repeats above, but I think you mean c.AdminThreshold here, again <= 0 comment applies